src/main/java/uk/gov/ons/census/action/model/entity/Case.java

```diff
@@ -92,4 +92,7 @@
 
   @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
   private boolean undeliveredAsAddressed;
+
+  @Column(columnDefinition = "BOOLEAN DEFAULT false")
```

Comment:
I feel like this should either follow the convention of explicitly setting `nullable = false` as well or completely do away with column definition if it isn't needed (I thought at one point we needed it so that hibernate didn't complain when in `verify` mode?)

Reply:
I'll follow the convention in the class for now 👍